oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/oidc/authentication/RegisteredClientOidcClientRegistrationConverter.java

```diff
@@ -31,7 +31,7 @@
  * @author Joe Grandja
  * @since 0.4.0
  */
-final class RegisteredClientOidcClientRegistrationConverter implements Converter<RegisteredClient, OidcClientRegistration> {
+public class RegisteredClientOidcClientRegistrationConverter implements Converter<RegisteredClient, OidcClientRegistration> {
```

Comment:
Make the class `final`. **NOTE:** This can still be reused using a delegation-based strategy.